core/src/main/resources/pl/messages.properties

```
@@ -102,7 +102,7 @@ wave.help.description=Pomachaj se
 wave.help.uzycie=<osoba>
 wave.help.extended=!<pusto>!
 
-starcatch.help.description=Wyskocz z okna po zdjęcie! W tej komendzie dostępne są flagi, przejrzyj je pod "%shelp starcatch"
+starcatch.help.description=Wyskocz z okna po zdjęcie! W tej komendzie dostępne są flagi, przejrzyj je pod "starcatch --help"
```

Prefix mogłeś zostawić imo
```suggestion
starcatch.help.description=Wyskocz z okna po zdjęcie! W tej komendzie dostępne są flagi, przejrzyj je pod "%sstarcatch --help"
```